qiita_pet/util.py

```diff
@@ -68,6 +69,15 @@ def clean_str(item):
     return str(item).replace(" ", "_").replace(":", "")
 
 
+def convert_text_html(message):
+    """Linkify URLs and turn newlines into <br/> for HTML"""
+    lines = message.split('\n')
```

Comment:
I think this entire function can be replaced by:
`return tornado_linkify(message.replace('\n', '<br/>'))`